src/System.Management.Automation/engine/remoting/commands/StartJob.cs

```diff
@@ -601,7 +601,7 @@ protected override void BeginProcessing()
                 ThrowTerminatingError(errorRecord);
             }
 
-            if (WorkingDirectory != null && !Directory.Exists(WorkingDirectory))
+            if (WorkingDirectory != null && !InvokeProvider.Item.IsContainer(WorkingDirectory))
```

Comment:
Please fix failing test for whitespace working directory.

Reply:
The test was fixed.